bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -230,14 +255,60 @@ pub mod pallet {
 				.saturating_add(T::BackendWeightInfo::transact_add_tip())
 		)]
 		pub fn add_tip(origin: OriginFor<T>, message_id: MessageId, asset: Asset) -> DispatchResult {
```

Comment:
If the asset is ETH (native eth), then we should bypass the swap and burn directly.